modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -206,7 +206,7 @@ export default {
 
       const result = await apos.modal.execute(editor, {
         schema: this.field.schema,
-        id: item._id,
+        id: item.attachment._id,
```

Comment:
We cannot hardcode the assumption that every document has an `attachment` property. I think you are using this to figure out which attachment is being cropped. In which case passing all of `item` here would provide the needed flexibility and remove assumptions that break relationship editors for other types that are not images.

Reply:
@boutell yes indeed, that's totally true, now that I see it I realise that's a dumb mistake.
I'll pass the whole item to keep this relationship component agnostic of the its properties